### PR TITLE
First idiom version

### PR DESCRIPTION
**Pull request by niknicius**

### CODE REVIEW

File: idioms/angularNestJsLambda/backend/models/entity.module.ts.example [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common';
+
+{{Entity module imports files}}
+
+@Module({
+  imports: [{{Entity module imports}}],
```
Comment (rodrigovilar) [inline idioms/angularNestJsLambda/backend/models/entity.module.ts.example]: Put the idiom file name here.
E.g.: Entity module imports - ./entity.module.imports.idiom
All idiom files must use .idiom extension.

File: idioms/angularNestJsLambda/backend/models/entity/entity.ts.model [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import {Column, DataType, Model, Table} from 'sequelize-typescript';
+
+@Table
+export class {{Entity singular name}} extends Model<{{Entity singular name}}> {
+
+    {{Entity attributes}}
```
Comment (rodrigovilar) [inline idioms/angularNestJsLambda/backend/models/entity/entity.ts.model]: ` {{For each Entity attributes: ./entity.model.attribute.*.idiom}}`
This wildcard should be used to several options of idioms in this place.

File: idioms/angularNestJsLambda/backend/models/entity/entity.attribute.options.model [hunk 3]
```diff
@@ -0,0 +1,4 @@
+@Column({
+   {{Attribute column options}}
```
Comment (rodrigovilar) [inline idioms/angularNestJsLambda/backend/models/entity/entity.attribute.options.model]: `{{Attribute column options - ./column.options/*.idiom}}`

File: idioms/angularNestJsLambda/backend/models/entity/options/auto-increment.idiom [hunk 4]
```diff
@@ -0,0 +1 @@
+@AutoIncrement
```
Comment (rodrigovilar) [inline idioms/angularNestJsLambda/backend/models/entity/options/auto-increment.idiom]: We will need to review all the options idioms.
They are at a very low level.
It is better to use more business-level concepts such as a primary key idiom with both autoincremet and primarykey decorators.

File: idioms/angularNestJsLambda/backend/models/resolver/entity.resolver.ts.idiom [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import {Args, Mutation, Query, Resolver} from '@nestjs/graphql';
+import { {{Entity singular name}} } from './model/{{Entity singular name}}.entity';
+import { {{Entity singular name}}Type } from './dto/{{Entity name}}.dto';
+import { {{Entity singular name}}Service } from './{{Entity name}}.service';
+
+@Resolver('{{Entity singular name}}')
+export class {{Entity singular name}}Resolver {
+    constructor(private readonly {{Entity  name}}Service: {{Entity singular name}}Service) {}
+
+    {{Resolver methods}}
```
Comment (rodrigovilar) [inline idioms/angularNestJsLambda/backend/models/resolver/entity.resolver.ts.idiom]: Do you need another idiom just for these methods?
Do they vary?

File: idioms/angularNestJsLambda/backend/models/resolver/entity.resolver.ts.idiom [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    @Query(() => {{Entity singular name}}Type)
+    async getOne{{Entity singular name}}(@Args('id') id: number) {
+        return await this.{{Entity name}}Service.find(id);
+    }
+
+    @Mutation(() => {{Entity singular name}}Type)
+    async create{{Entity singular name}}(@Args('input') input: Create{{Entity singular name}}Input): Promise<{{Entity singular name}}> {
+        return await this.bookService.create(book);
```
Comment (niknicius) [inline idioms/angularNestJsLambda/backend/models/resolver/entity.resolver.ts.idiom:22]: Static entity instead of dynamic

**Pull request merged**